flax/training/checkpoints.py

```diff
@@ -111,6 +111,8 @@ def save_checkpoint(ckpt_dir: Union[str, os.PathLike],
     checkpoint_files.append(ckpt_path)
 
   checkpoint_files = natural_sort(checkpoint_files)
+  if checkpoint_files[-1] == ckpt_tmp_path:
```

Comment:
Can we add a short comment here explaining what this check does? (Basically the contents of this PR)? So that future collaborators on this code don't have to `git blame` to find this PR?

Reply:
I have done this now

Reply:
Thanks Anurag!